openedx/stanford/lms/urls.py

```diff
@@ -0,0 +1,41 @@
+from django.conf import settings
+from django.conf.urls import url
+
+
+urlpatterns = (
+    url(
+        r'^course_sneakpeek/{}/$'.format(
+            settings.COURSE_ID_PATTERN,
+        ),
+        'student.views.setup_sneakpeek',
+        name='course_sneakpeek',
+    ),
+    url(
```

Comment:
is the change in ordering just alphabetization?

Reply:
Yup.